### DISCUSSION TITLE
PR Review file header does not stay on screen

### DISCUSSION THREAD

**Discussion by ModProg** [community/community#11781] [Mobile] [answered]
This makes it really hard to mark files as seen, when you have scrolled down.

I would expect this to stay on screen when scrolling down: 

![Screenshot_20220219-190243_GitHub.jpg](https://user-images.githubusercontent.com/11978847/154813357-32079ccf-06d8-48a9-8a39-685b1dacecca.jpg)

But it doesn't:

![Screenshot_20220219-190303_GitHub.jpg](https://user-images.githubusercontent.com/11978847/154813380-e085737c-1377-4a4c-99fe-2c011a7ea033.jpg)

**Comment by aconsuegra** [comment, marked as answer]
Hey @ModProg, thanks for your feedback! 
I created an issue for our engineering team to track this enhancement. I hope we can include it in the app soon.

**Comment by FakieG** [comment]
New features on the way
